src/main/java/slimeknights/tconstruct/library/Util.java

```diff
@@ -196,4 +198,9 @@ public static boolean clickedAABB(AxisAlignedBB aabb, float hitX, float hitY, fl
             && aabb.minZ <= hitZ && hitZ <= aabb.maxZ;
   }
 
+  public static String toIdentedStringList(Collection<?> list) {
```

Comment:
While overall it would be nice to Javadoc everything except overrides, I specifically like the idea of javadocs for everything in util.

Reply:
Will do after merge